source/templates.tex

```diff
@@ -1390,16 +1388,14 @@
   T& elem(int i) { return v[i]; }
 };
 \end{codeblock}
-
-\pnum
 The prefix
 \tcode{template}
 \tcode{<class}
 \tcode{T>}
 specifies that a template is being declared and that a
 \grammarterm{type-name}
 \tcode{T}
-will be used in the declaration.
+will be used in the definition.
```

Comment:
@timsong-cpp: Yes, but not all declarations are definitions. To repeat, I think the example should be *useful*, otherwise we can just remove it. This change is an attempt to make the sentence relevant to the example and help the illustraion, rather than be a general statement that happens to be true.